### ISSUE TITLE
iFrame titles should be translatable

### ISSUE THREAD

**Issue by ojeytonwilliams** [freeCodeCamp/freeCodeCamp#45050]
Now that https://github.com/freeCodeCamp/freeCodeCamp/pull/45014/ is in, we're adding titles of the form 'CatPhotoApp preview' and 'preview'.  This is good from an a11y perspective, but the string needs to be translatable.

**Comment by Nirajn2311** [comment]
Apply a `lang` attribute to `iframe` element
